src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -621,9 +621,10 @@ public async Task<IActionResult> ResumeApprenticeship(ResumeRequestViewModel vie
         [HttpGet]
         [DasAuthorize(CommitmentOperation.AccessApprenticeship)]
         [Route("{apprenticeshipHashedId}/details", Name = RouteNames.ApprenticeDetail)]
-        public async Task<IActionResult> ApprenticeshipDetails(ApprenticeshipDetailsRequest request)
+        public async Task<IActionResult> ApprenticeshipDetails(ApprenticeshipDetailsRequest request, bool showPriceChangeRejected)

```

Comment:
Looks like the unit test for this method needs updating following adding this parameter